sdk/maps/azure-maps-geolocation/tests/test_client_unit.py

```diff
@@ -0,0 +1,69 @@
+import sys
+import pytest
+try:
+    from unittest.mock import Mock
+except ImportError:  # python < 3.3
+    from mock import Mock  # type: ignore
+
+from devtools_testutils import AzureTestCase
+from azure.core.pipeline.transport import HttpTransport, HttpResponse
+from azure.core.credentials import AzureKeyCredential
+from azure.maps.geolocation import MapsGeolocationClient
+
+
+
+# cSpell:disable
+class MockTransport(HttpTransport):
+    def __init__(self, status_code, body, **kwargs):
+        self.status_code = status_code
+        self.body = body.encode("utf-8-sig") if body != None else None
+        self.kwargs = kwargs
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        pass
+    def close(self):
+        pass
+    def open(self):
+        pass
+    def send(self, request, **kwargs):  # type: (PipelineRequest, Any) -> PipelineResponse
+        response = HttpResponse(request, None)
+        response.status_code = self.status_code
+        response.headers["content-type"] = "application/json"
+        response.body = lambda: self.body
+        for key, val in self.kwargs.items():
+            setattr(response, key, val)
+        return response
+
+def create_mock_client(status_code=0, body=None, **kwargs):
+    return MapsGeolocationClient(credential= Mock(AzureKeyCredential),
+                        transport=MockTransport(status_code, body, **kwargs))
+
+class AzureMapsGeolocationClientUnitTest(AzureTestCase):
+    def __init__(self, *args, **kwargs):
+        super(AzureMapsGeolocationClientUnitTest, self).__init__(*args, **kwargs)
+
+    def setUp(self):
+        super(AzureMapsGeolocationClientUnitTest, self).setUp()
+
+
```

Comment:
Suggest to leave only one empty line within class methods. See [PEP-0008](https://peps.python.org/pep-0008/#blank-lines)